lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1285,6 +1285,7 @@ def set_log_entry_fields(record, entry)
     # Translates other severity strings to one of the valid values above.
     SEVERITY_TRANSLATIONS = {
       # log4j levels (both current and obsolete).
+      'SEVERE' => 'ERROR',
```

Comment:
`SEVERE` does not seem to be a standard level for log4j: https://www.tutorialspoint.com/log4j/log4j_logging_levels.htm

Maybe it could belong to "other misc".

Also, why is `SEVERE` mapped to `ERROR` instead of `CRITICAL`?

Reply:
This is from [`java.util.logging`](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html#SEVERE), but +1 on moving it into a separate section below (right before `# nginx levels ...`).

Reply:
Agreed. We could add a special case for this so as to not confuse them with log4j levels.